### PR TITLE
feat/ basic css set up

### PR DESCRIPTION
**Pull request by brendel-r**
inital set up for css

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.header-container {
+  display: flex;
+  flex-direction: row;
+  margin: 1%;
+  justify-content: space-around;
+}
+
+#header-logo {
+  border-radius: 150px;
```
Comment (CapCinematic) [inline src/components/Header/Header.css:9]: CSS LOOKS GREAT! Very neat and fluid.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import logo from '../../Assets/glam-logo.png'
...
 
   return (
-    <header>
-      <h1>
-        <img src= {logo} id='header-logo' alt="Glam-Squad" />
-        Glam-Squad
+    <header className='header-container'>
+      <img src= {logo} id='header-logo' alt="Glam-Squad logo" />
+      <h1 className='slogan'>
+        Elevate your beauty game with Glam Squad!
```
Comment (Devynnem) [inline src/components/Header/Header.tsx:12]: Love the pulsing title!

**Review by CapCinematic — APPROVED** [review]
Let's handle these conflicts together.

**Review by tdmburr — APPROVED** [review]
This looks great, we'll work on getting it implemented!

**Review by Devynnem — APPROVED** [review]
Woo!

**Pull request merged**